internal/backend/dispatcher/special_triggers.go

```diff
@@ -0,0 +1,64 @@
+package dispatcher
+
+import (
+	"errors"
+
+	httpint "go.autokitteh.dev/autokitteh/integrations/http"
```

Comment:
Why are you aliasing this package? I don't see any other instance of "http"

Reply:
`http` is confusing, this makes sure it's not.